RomanPort.PCParadiseBot/Entities/IDiscordFilter.cs

```diff
@@ -27,11 +25,12 @@ private async Task Discord_MessageCreated(MessageCreateEventArgs e)
                 //Check if supported
                 if (await IsAccepted(e))
                 {
-                    OnAccepted(e);
+                    await OnAccepted(e);
```

Comment:
Awaiting this causes a lock.

Reply:
Can we use await Task.Run() instead? that would run it in it's own background thread which would avoid the locking, and would get rid of the warnings due to it running synchronously - which may cause issue in the future.

Reply:
Added a commit to show what I'm talking about.